skills/cryptocompare/fetch_news.py

```diff
@@ -0,0 +1,31 @@
+"""Tool for fetching news via CryptoCompare API."""
+
+import asyncio
+from typing import Any, Dict, Type
+from pydantic import BaseModel
+from skills.cryptocompare.base import CryptoCompareBaseTool
+from skills.cryptocompare.api import fetch_news, FetchNewsInput
+
+class CryptoCompareFetchNewsOutput(BaseModel):
+    result: Dict[str, Any]
+    error: str | None = None
+
+class CryptoCompareFetchNews(CryptoCompareBaseTool):
+    name: str = "cryptocompare_fetch_news"
+    description: str = "Fetch cryptocurrency news for a given token using CryptoCompare API"
+    args_schema: Type[BaseModel] = FetchNewsInput
+
+    def _run(self) -> CryptoCompareFetchNewsOutput:
+        raise NotImplementedError("Use _arun instead")
+
+    async def _arun(self) -> CryptoCompareFetchNewsOutput:
```

Comment:
There is the real error, _arun must have the same params as FetchNewsInput. Every function has the same issue.